src/main/java/com/openlattice/postgres/PostgresTable.java

```diff
@@ -212,6 +212,21 @@
                             ANALYZER,
                             MULTI_VALUED,
                             INDEX_TYPE );
+
+    public static final PostgresTableDefinition FLIGHTS =
```

Comment:
Don't do this. This object is complex enough that you'll just want to store it as JSON, especially if you're accessing via a mapstore.